azkaban-common/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
   compile('io.dropwizard.metrics:metrics-core:3.1.0')
   compile('io.dropwizard.metrics:metrics-jvm:3.1.0')
 
+  testCompile files(System.getenv("JAVA_HOME") + "/db/lib/derby.jar")
```

Comment:
@HappyRay Checked google resourced. Even though JDK comes with this jar, Java does not rely on it when being launched. So I come up this work around. What do you think?